### ISSUE TITLE
[Feature]Checking of number of tokens present in user or organizations open api key''s tokens

### ISSUE THREAD

**Issue by litesh1123** [Chit-Chat-Org/chit-chat-nextjs#29]
**Is your feature request related to a problem? Please describe.**
As chatgpt open api key has number of tokens assigned and evey prompt given content generated  tokens are used.
As in this project open api key plays a crucial role ,user/org must be able to check how many tokens present in there api key
So I propose section where user will be able to check tokens in there respective key

**Describe the solution you'd like**
Work must be done over two things
>fronted: Addition of section for checking tokens
>backend :logic need to be created

**Comment by litesh1123** [comment]
@apurvjha123 please let me know how is the idea and please give your feedback on it

**Comment by litesh1123** [comment]
These are huge features need to work on frontend as well as backend so would be creating more issues on these ,please let me know how are my ideas related to improvement, enhancement in this project

**Comment by apurvjha123** [comment]
@litesh1123 this is really good feature and necessary for this website.

**Comment by litesh1123** [comment]
@apurvjha123 thank you mam , I have gone through project it's great idea  I have brain stormed and suggested these features as this would enhance project and also would be great user experience

**Comment by apurvjha123** [comment]
Exactly, many of the contributor just spam for updating readme and ci cd but You went through the project.

Really appreciate !

**Comment by litesh1123** [comment]
@apurvjha123 thank you for considering my suggestions actually that video in readme was great decision I have gone through video got idea .

I have been trying to connect you but couldn't find linkedin. 

Please give me some time I Will work and update

**Comment by apurvjha123** [comment]
Connected!

Start working. If you need any help related to project just feel free to ask me !

**Comment by litesh1123** [comment]
Thank you will let u know if anything needed

**Comment by apurvjha123** [comment]
Happy Hacking !

**Comment by Shivani756** [comment]
Hello sir@apurvjha123 
I would request you allow me and @litesh1123 to work together so that I could also incorporate my suggestions
so i would request you to assign me also this issue

**Comment by litesh1123** [comment]
@apurvjha123 if possible please assign @Shivani756 so that we can collab on this issue
